README.md

```diff
@@ -40,54 +40,101 @@ git clone https://github.com/fortran-lang/stdlib
 cd stdlib
 ```
 
+
 ### Requirements
 
-The preprocessor ```fypp``` (https://github.com/aradi/fypp) is needed because metaprogramming is used.
-It can be installed using the command line installer ```pip```.
+To build the Fortran standard library you need
+
+- a Fortran 2008 compliant compiler, or better, a Fortran 2018 compliant compiler
+  (GCC Fortran and Intel Fortran compilers are known to work for stdlib)
```

Comment:
Should a minimum version for both compilers be mentioned? E.g., GFortran 7 does not work properly  (in debug mode) with stdlib.

Reply:
I rolled back my Intel version and checked from 16.0 to 19.0, bad news is that only 19.0 works for stdlib.

Reply:
> I rolled back my Intel version and checked from 16.0 to 19.0, bad news is that only 19.0 works for stdlib.

A way to avoid to mention the lowest version of compiler could be to list the compilers that are tested in the CI (e.g., GFortran 8,9,10, and Intel OneAPI). For these compilers we are always sure it works properly.

Reply:
Added a table with supported compilers as well as known failing compilers